react/components/utils.ts

```diff
@@ -0,0 +1,46 @@
+export const normalizeValue = (value: number, maxValue?: number) => {
+  return maxValue ? Math.min(value, maxValue) : value
+}
```

Comment:
What do you think of removing this function and always use `Math.min(value, maxValue ?? value)` instead?

- The `normalizeValue` name looks confusing to me, as "normalize" can mean a lot of different things, and as far as I know, the meaning here is not one of the most common ones
- Using `Math.min` is only a bit more verbose
- It's very explicit
- Can be used as `Math.min(value, maxValue)` when the `maxValue` is a `number`

It's a smol thing, but I think it would improve the readability.

Reply:
sure, will do